src/main/java/com/synopsys/integration/jenkins/detect/substeps/DetectJarManager.java

```diff
@@ -77,15 +73,15 @@ public String calculateJavaExecutablePath() throws IOException {
         return javaExecutablePath;
     }
 
-    public void logJavaVersion() throws InterruptedException {
-        logger.debug("PATH: " + environmentVariables.get("PATH"));
-        if (LogLevel.DEBUG == logger.getLogLevel()) {
+    private void logDebugData(String javaExecutablePath) throws InterruptedException {
+        if (logger.getLogLevel().isLoggable(LogLevel.DEBUG)) {
             try {
-                logger.info("Java version: ");
-                ProcessBuilder processBuilder = new ProcessBuilder(Arrays.asList("java", "-version"));
+                logger.debug("PATH: " + environmentVariables.get("PATH"));
```

Comment:
Although PATH isn't used, this could be beneficial should failures occur at runtime, so I left this in.